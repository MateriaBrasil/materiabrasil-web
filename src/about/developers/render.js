import React from 'react'

import Colors from '../../Colors'
import Session from '../Session'
import Members from '../Members'
import Title from '../Title'
import Subtitle from '../Subtitle'
import Paragraph from '../Paragraph'
import developersList from './developersList'

export default () => (
  <Session id="developers">
    <Title color>Desenvolvedores</Title>
    <Paragraph color>
      Para o desenvolvimento desses direcionadores a MateriaMundi contou com uma
      equipe de consultores especialistas em diversas áreas.
    </Paragraph>
    <Subtitle>Conheça o nosso time!</Subtitle>
    <Members list={developersList} color={Colors.black} />
  </Session>
)
