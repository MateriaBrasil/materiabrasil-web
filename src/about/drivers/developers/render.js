import React, { Fragment } from 'react'

import Members from '../../Members'
import Subtitle from '../../Subtitle'
import Paragraph from '../../Paragraph'
import membersList from './developersList'

export default () => {
  return (
    <Fragment>
      <Subtitle>Desenvolvedores</Subtitle>
      <Paragraph>
        Para o desenvolvimento desses direcionadores a MateriaMundi contou com
        uma equipe de consultores especialistas em diversas áreas.
      </Paragraph>
      <Paragraph>Conheça o nosso time!</Paragraph>
      <Members membersList={membersList} />
    </Fragment>
  )
}
