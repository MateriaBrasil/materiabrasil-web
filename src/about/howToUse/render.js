import React from 'react'

import Section from '../Section'
import Title from '../Title'
import Video from '../../Video'
import VideoStyle from './VideoStyle'

export default () => (
  <Section
    id="how_to_use"
    backgroundImage="url('https://s3.amazonaws.com/materiamundi-us/static/backgrounds/material-3.jpg')"
  >
    <Title>Como usar</Title>
    <VideoStyle>
      <Video />
    </VideoStyle>
  </Section>
)
