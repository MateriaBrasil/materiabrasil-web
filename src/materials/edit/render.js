import React from 'react'

import Edit from './Edit'

export default props => editProps => <Edit {...props} {...editProps} />
