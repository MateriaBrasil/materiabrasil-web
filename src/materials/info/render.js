import React from 'react'
import FragmentWithHelmet from '../../FragmentWithHelmet'

import Info from './Info'

export default props => (info, infoProps) => {
  return (
    <FragmentWithHelmet
      title={infoProps.info.name}
      ogImage={infoProps.info.listImageUrl}
    >
      <Info current={info} {...props} {...infoProps} />
    </FragmentWithHelmet>
  )
}
