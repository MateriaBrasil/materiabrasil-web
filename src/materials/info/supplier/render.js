import React from 'react'

import Info from './Info'

export default props => (info, infoProps) => (
  <Info supplier={info} {...props} {...infoProps} />
)
