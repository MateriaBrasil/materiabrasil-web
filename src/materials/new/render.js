import React from 'react'

import Form from '../form/Form'
import createWithParams from './createWithParams'

export default props => ({ create, creating, error }) => {
  const { match } = props
  const { id } = match.params

  return (
    <Form
      onSubmit={createWithParams(create, id)}
      submitting={creating}
      createError={error}
      buttonText="Cadastrar material"
      {...props}
    />
  )
}
