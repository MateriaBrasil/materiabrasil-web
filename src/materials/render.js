import React from 'react';
import Grid from '@material-ui/core/Grid';
import Router from './Router';
import NavBarAndContent from '../NavBarAndContent';
import Header from './info/header/index';

export default props => routeProps => {
  // if (false) {
  //   return (
  //     <Grid style={{ backgroundColor: '#fff' }}>
  //       <Header>
  //         <NavBarAndContent style={{ zIndex: '10' }} transparent {...props} />
  //       </Header>
  //       <Router {...props} />
  //     </Grid>
  //   );
  // } else {
  return (
    <Grid style={{ backgroundColor: '#fff' }}>
      <Router {...props} />
    </Grid>
  );
  // }
};
