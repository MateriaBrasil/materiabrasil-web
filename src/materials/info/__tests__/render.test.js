import React from 'react'
import renderer from 'react-test-renderer'

import render from '../render'

jest.mock('../Info', () => props => <div {...props}>Info</div>)

it('renders correctly', () => {
  const props = {
    infoProps: {
      info: {
        name: 'foo-name',
        listImageUrl: 'list-url',
      },
    },
    bar: 'foo',
  }
  const tree = renderer
    .create(render(props)(props.infoProps.info, props.infoProps))
    .toJSON()
  expect(tree).toMatchSnapshot()
})
