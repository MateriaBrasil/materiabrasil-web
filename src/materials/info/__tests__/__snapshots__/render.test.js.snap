// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  bar="foo"
  current={
    Object {
      "bar": "foo",
    }
  }
  info={
    Object {
      "listImageUrl": "list-url",
      "name": "foo-name",
    }
  }
>
  Info
</div>
`;
